src/assets/translations/cs-CS.json

```
@@ -2659,7 +2659,7 @@
       "Provisioning Criteria": "Kritéria poskytování",
       "Purchase Price Ratio": "Poměr kupní ceny",
       "Purchased/Redeemed Price": "Kupovaná/uplatněná cena",
```

Biome (2.4.15)

[error] The key Purchased/Redeemed Price was already declared.

(lint/suspicious/noDuplicateObjectKeys)